### DISCUSSION TITLE
Allowlist/denylist language

### DISCUSSION THREAD

**Discussion by popaprozac** [supabase/supabase#1033] [Questions] [answered]
Hello!

Just spun up my first project and noticed in Authentication -> Settings there is a reference to "whitelisting". More and more we are moving towards better language like allowlist and denylist for the same meaning. I suggest this is changed to allowlist.

> The base URL of your website. Used as a whitelist for redirects and for constructing URLs used in emails.

**Comment by J0** [comment, marked as answer]
Hey,

It's been a while since this was posted -- thanks for flagging this! We've amended the page a while ago and we've also just removed all occurrences of "whitelist/blacklist" in our docs.
